ldap_auth_provider.py

```diff
@@ -404,6 +404,74 @@ class _LdapConfig(object):
 
         return ldap_config
 
+    def _get_server(self, get_info: Optional[str] = None) -> ldap3.ServerPool:
+        """Constructs ServerPool from configured LDAP URIs
+
+        Args:
+            get_info (str, optional): specifies if the server schema and server
+            specific info must be read. Defaults to None.
+
+        Returns:
+            [ServerPool]: servers groupped to ServerPool
```

Comment:
no need to repeat the information from the type annotation here.

```suggestion
            servers grouped in a ServerPool
```